internal/service/alertconfiguration/data_source_alert_configuration.go

```diff
@@ -1,22 +1,24 @@
-package mongodbatlas
+package alertconfiguration
 
 import (
 	"context"
 	"fmt"
+	"sort"
 
 	"github.com/hashicorp/hcl/v2/hclwrite"
 	"github.com/hashicorp/terraform-plugin-framework-validators/stringvalidator"
 	"github.com/hashicorp/terraform-plugin-framework/datasource"
 	"github.com/hashicorp/terraform-plugin-framework/datasource/schema"
 	"github.com/hashicorp/terraform-plugin-framework/schema/validator"
 	"github.com/hashicorp/terraform-plugin-framework/types"
+	"github.com/mongodb/terraform-provider-mongodbatlas/internal/config"
 	"github.com/mongodb/terraform-provider-mongodbatlas/mongodbatlas/util"
 	"github.com/zclconf/go-cty/cty"
 	"go.mongodb.org/atlas-sdk/v20231115001/admin"
 )
 
-var _ datasource.DataSource = &AlertConfigurationDS{}
-var _ datasource.DataSourceWithConfigure = &AlertConfigurationDS{}
+var _ datasource.DataSource = &DS{}
```

Comment:
nit: I don't know team feeling but I am not a fan of `DS` and `RS` - since names are not the long I would hope if we can extend them for better readability.

Reply:
it was because a linter issue but I can definitely find a better name, i'm not convinced by DS either:

exported: type name will be used as alertconfiguration.AlertConfigurationRS by other packages, and that stutters; consider calling this RS.

Reply:
I've changed it making them private, alertConfigurationDS and alertConfigurationRS, as they're only needed inside the alertconfiguration package.